src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -285,6 +287,14 @@ public void updateUserSyncTcfBlockedMetric(String bidder) {
         userSync().forBidder(bidder).tcf().incCounter(MetricName.blocked);
     }
 
+    public void updateUserSyncTcfInvalidMetric(String bidder) {
+        userSync().forBidder(bidder).tcf().incCounter(MetricName.invalid);
+    }
+
+    public void updateUserSyncTcfInvalidMetricForAllBidders() {
```

Comment:
This method can overloaded from `updateUserSyncTcfInvalidMetric` but without args.
Also, it call `updateUserSyncTcfInvalidMetric(ALL_REQUEST_BIDDERS)` directly.